src/core/flow.js

```diff
@@ -44,12 +47,23 @@ export function Flow({
   validator,
   processMove,
   optimisticUpdate,
+  canMakeMove,
 }) {
   if (!ctx) ctx = () => ({});
   if (!events) events = {};
   if (!init) init = state => state;
   if (!validator) validator = () => true;
   if (!processMove) processMove = state => state;
+  if (!canMakeMove) {
+    canMakeMove = (G, ctx, playerID) => {
+      const actionPlayers = ctx.actionPlayers || [];
+      return (
+        playerID === ctx.currentPlayer ||
```

Comment:
Sometimes you may not want the *currentPlayer* to play (even if they own the turn). Let's just use *actionPlayers* here.

Reply:
OK.